### DISCUSSION TITLE
how can i edit the hacks it checks for?

### DISCUSSION THREAD

**Discussion by Uncommon-34** [Scythe-Anticheat/Scythe-Anticheat#89] [Help and Support] [answered]
i want to be able to change what hacks it looks for as I'm trying to use items with names over 32 digits and so I want to turn that off but it says the method you give is outdated and there is no file called config.js. any support is appreciated thank you

**Comment by MrDiamond64** [comment, marked as answer]
Config.js should be located in the pack folder->scripts->data.

**Reply by Uncommon-34** [reply]
thank you
